lib/crawly/pipelines/experimental/send_to_ui.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Crawly.Pipelines.Experimental.SendToUI do
+  @moduledoc """
+  """
+  @behaviour Crawly.Pipeline
+
+  require Logger
+
+  @impl Crawly.Pipeline
+  def run(item, state, opts \\ []) do
+    job_tag =
+      case Map.get(state, :job_tag, nil) do
+        nil ->
+          UUID.uuid1()
+
+        tag ->
+          tag
+      end
+
+    ui_node =
+      case Keyword.get(opts, :ui_node) do
+        nil ->
+          throw(
```

Comment:
Instead of throwing, it would be better to gracefully do nothing and log errors instead.

Throwing would cause the entire pipe to fail

Reply:
👍